protobuf/sparkplug_b.proto

```diff
@@ -0,0 +1,224 @@
+// * Copyright (c) 2015, 2018 Cirrus Link Solutions and others
+// *
+// * This program and the accompanying materials are made available under the
+// * terms of the Eclipse Public License 2.0 which is available at
```

Comment:
I don't know how this works. Can you commit to our Apache licensed software a different one?